basxconnect/core/views/person/person_browse_views.py

```diff
@@ -1,20 +1,28 @@
+import bread
+import django.conf
 import htmlgenerator as hg
 from bread import layout as layout
+from bread import menu
 from bread.layout.components.datatable import DataTableColumn
+from bread.layout.components.modal import Modal, modal_with_trigger
 from bread.utils import get_concrete_instance
-from bread.utils.links import Link
-from bread.utils.urls import reverse
+from bread.utils.links import Link, ModelHref
+from bread.utils.urls import reverse, reverse_model
 from bread.views import BrowseView, BulkAction
 from bread.views.browse import delete as breaddelete
 from bread.views.browse import export as breadexport
 from bread.views.browse import restore as breadrestore
+from build.lib.bread.layout.components.button import Button
```

Comment:
This is a strange import, see CI messages for this.

Reply:
oops, sorry. I should be more careful with IDE features :D